cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -84,6 +84,31 @@ The CLI may be started with command-line options, as detailed below::
    -u,--uri <arg>           CDAP instance URI to interact with in the format
                             "[http[s]://]<hostname>[:<port>[/<namespace>]]".
                             Defaults to "http://127.0.0.1:10000/default".
+                            
+
+Settings
+--------
+
+Certain commands (``connect`` and ``render``) affect how CLI works for the duration of a session.
+
+The command ``"cli render as <table-renderer>"`` sets how table data is rendererd. Valid options are
+either ``"alt"`` (the default) and ``"csv"``.
+
+- With ``"cli render as alt"`` (the default), a command such as ``"list apps"`` will be output as::
+
+    +================================+
+    | app id      | description      |
+    +=============+==================+
+    | PurchaseApp | Some description |
+    +=============+==================+
+
+- With ``"cli render as csv"``, the same ``"list apps"`` would be output as::
```

Comment:
Maybe also add why they would use this. (It's b/c `alt` may split a cell into multiple lines, so it's not possible to copy paste)